frog/imports/internalActivities/ac-quiz/utils.js

```diff
@@ -78,6 +78,23 @@ export const computeProgress = (
   return nAnsweredQuestions / questions.length;
 };
 
+export const isCorrect = (
+  qData: { [idx: string]: boolean, text: string },
+  qConfig: { answers: Object[], text: boolean }
+) => {
+  return !qConfig.answers.some((x, i) => !!x.isCorrect !== !!qData[`${i}`]);
```

Comment:
I'm trying to parse this line, but find it hard. Why are you turning both x.isCorrect and qData... into booleans before comparing them?

Reply:
`isCorrect ` is either false of undefined for wrong answers (configured by teachers)
`qData[`${i}`]` is either false or undefined for not selected answers (by students from ActivityRunner) 

They are undefined by default and take value false only if for example the user selects, then unselects the answer or the teacher ticks then unticks the configuration box.

`!qConfig.answers.some((x, i) => !!x.isCorrect !== !!qData[`${i}`])` 
It says if there is "some" in the config answers for which the boolean "isCorrect" is not the same as the boolean "reactive data" then return false

Thanks for always reminding me to add tests. You're completely right :)